Upgrade Matter SDK to v1.1.0.1

MonicaisHer opened the pull request:
For the `chip-tool pairing` command, the `ethernet` subcommand has been removed, and the following six new commands have been added in the latest version v1.1.0.1:
```
 already-discovered                                                                
 already-discovered-by-index                                                       
 already-discovered-by-index-with-wifi      
 get-commissioner-node-id                                                          
 get-commissioner-root-certificate  
 issue-noc-chain       
 ```
 #### Test
 The snap built from this PR has already been tested for pairing and controlling the matter-bridge-tapo-light snap using:
 ```
 sudo chip-tool pairing onnetwork 110 20202021
 sudo chip-tool onoff toggle 110 1
 ```

farshidtz commented:
Thanks. Could you also test with the [matter-pi-gpio-commander](https://snapcraft.io/matter-pi-gpio-commander) which is built directly on the C++ SDK?

MonicaisHer commented:
The chip-tool built by Snap from this PR has some difficulties in pairing with the matter-gpio-commander. The logs are as follows:
```
$ sudo chip-tool pairing onnetwork 110 20202021
[1686666964.075492][20463:20463] CHIP:DL: ChipLinuxStorage::Init: Using KVS config file: /mnt/chip_kvs
[1686666964.075749][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_kvs-Bbk0VG)
[1686666964.075790][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_kvs)
[1686666964.076861][20463:20463] CHIP:DL: ChipLinuxStorage::Init: Using KVS config file: /mnt/chip_factory.ini
[1686666964.076882][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_factory.ini-u4CF6d)
[1686666964.076897][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_factory.ini)
[1686666964.076904][20463:20463] CHIP:DL: ChipLinuxStorage::Init: Using KVS config file: /mnt/chip_config.ini
[1686666964.076914][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_config.ini-gFdJEc)
[1686666964.076924][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_config.ini)
[1686666964.076928][20463:20463] CHIP:DL: ChipLinuxStorage::Init: Using KVS config file: /mnt/chip_counters.ini
[1686666964.076936][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_counters.ini-vHR2PF)
[1686666964.076945][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_counters.ini)
[1686666964.076965][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_factory.ini-1ZtKc5)
[1686666964.077023][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_factory.ini)
[1686666964.077026][20463:20463] CHIP:DL: NVS set: chip-factory/unique-id = "D190D5F3FCD0B017"
[1686666964.077040][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_factory.ini-xCJcwV)
[1686666964.078415][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_factory.ini)
[1686666964.078420][20463:20463] CHIP:DL: NVS set: chip-factory/vendor-id = 65521 (0xFFF1)
[1686666964.078436][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_factory.ini-t9ium7)
[1686666964.078469][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_factory.ini)
[1686666964.078473][20463:20463] CHIP:DL: NVS set: chip-factory/product-id = 32769 (0x8001)
[1686666964.078487][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_counters.ini-vdKPig)
[1686666964.078513][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_counters.ini)
[1686666964.078516][20463:20463] CHIP:DL: NVS set: chip-counters/reboot-count = 1 (0x1)
[1686666964.078531][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_counters.ini-y00g3b)
[1686666964.078561][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_counters.ini)
[1686666964.078565][20463:20463] CHIP:DL: NVS set: chip-counters/total-operational-hours = 0 (0x0)
[1686666964.078578][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_counters.ini-KH9sen)
[1686666964.078624][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_counters.ini)
[1686666964.078629][20463:20463] CHIP:DL: NVS set: chip-counters/boot-reason = 0 (0x0)
[1686666964.078653][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_config.ini-LB0tST)
[1686666964.078678][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_config.ini)
[1686666964.078681][20463:20463] CHIP:DL: NVS set: chip-config/regulatory-location = 0 (0x0)
[1686666964.078701][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_config.ini-uQ9ukY)
[1686666964.078758][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_config.ini)
[1686666964.078767][20463:20463] CHIP:DL: NVS set: chip-config/location-capability = 2 (0x2)
[1686666964.078975][20463:20463] CHIP:DL: Got Ethernet interface: eno1
[1686666964.079106][20463:20463] CHIP:DL: Found the primary Ethernet interface:eno1
[1686666964.079266][20463:20463] CHIP:DL: Failed to get WiFi interface
[1686666964.079268][20463:20463] CHIP:DL: Failed to reset WiFi statistic counts
[1686666964.080663][20463:20463] CHIP:IN: UDP::Init bind&listen port=0
[1686666964.080677][20463:20463] CHIP:IN: UDP::Init bound to port=51610
[1686666964.080679][20463:20463] CHIP:IN: UDP::Init bind&listen port=0
[1686666964.080686][20463:20463] CHIP:IN: UDP::Init bound to port=53278
[1686666964.080687][20463:20463] CHIP:IN: BLEBase::Init - setting/overriding transport
[1686666964.080689][20463:20463] CHIP:IN: TransportMgr initialized
[1686666964.080693][20463:20463] CHIP:FP: Initializing FabricTable from persistent storage
[1686666964.080698][20463:20463] CHIP:TS: Last Known Good Time: [unknown]
[1686666964.080701][20463:20463] CHIP:TS: Setting Last Known Good Time to firmware build time 2023-06-12T15:40:55
[1686666964.080847][20463:20463] CHIP:ZCL: Using ZAP configuration...
[1686666964.082385][20463:20463] CHIP:CTL: System State Initialized...
[1686666964.082394][20463:20463] CHIP:CTL: Stopping commissioning discovery over DNS-SD
[1686666964.082402][20463:20463] CHIP:CTL: Setting attestation nonce to random value
[1686666964.082405][20463:20463] CHIP:CTL: Setting CSR nonce to random value
[1686666964.082412][20463:20463] CHIP:IN: UDP::Init bind&listen port=5550
[1686666964.082420][20463:20463] CHIP:IN: UDP::Init bound to port=5550
[1686666964.082421][20463:20463] CHIP:IN: UDP::Init bind&listen port=5550
[1686666964.082426][20463:20463] CHIP:IN: UDP::Init bound to port=5550
[1686666964.082427][20463:20463] CHIP:IN: TransportMgr initialized
[1686666964.082461][20463:20503] CHIP:DL: CHIP task running
[1686666964.082475][20463:20503] CHIP:DL: HandlePlatformSpecificBLEEvent 32784
[1686666964.082531][20463:20503] CHIP:CTL: Stopping commissioning discovery over DNS-SD
[1686666964.082536][20463:20503] CHIP:CTL: Setting attestation nonce to random value
[1686666964.082552][20463:20503] CHIP:CTL: Setting CSR nonce to random value
[1686666964.082568][20463:20503] CHIP:CTL: Couldn't get ExampleOpCredsCAKey from storage: src/controller/ExamplePersistentStorage.cpp:93: CHIP Error 0x000000A0: Value not found in the persisted storage
[1686666964.082707][20463:20503] CHIP:CTL: Couldn't get ExampleOpCredsICAKey from storage: src/controller/ExamplePersistentStorage.cpp:93: CHIP Error 0x000000A0: Value not found in the persisted storage
[1686666964.082859][20463:20503] CHIP:CTL: Generating RCAC
[1686666964.083882][20463:20503] CHIP:CTL: Generating ICAC
[1686666964.083979][20463:20503] CHIP:CTL: Generating NOC
[1686666964.084100][20463:20503] CHIP:FP: Validating NOC chain
[1686666964.084497][20463:20503] CHIP:FP: NOC chain validation successful
[1686666964.085413][20463:20503] CHIP:FP: Added new fabric at index: 0x1
[1686666964.085416][20463:20503] CHIP:FP: Assigned compressed fabric ID: 0xA2C67A67B4C737DD, node ID: 0x000000000001B669
[1686666964.085419][20463:20503] CHIP:TS: Last Known Good Time: 2023-06-12T15:40:55
[1686666964.085420][20463:20503] CHIP:TS: New proposed Last Known Good Time: 2021-01-01T00:00:00
[1686666964.085421][20463:20503] CHIP:TS: Retaining current Last Known Good Time
[1686666964.085531][20463:20503] CHIP:FP: Metadata for Fabric 0x1 persisted to storage.
[1686666964.085735][20463:20503] CHIP:TS: Committing Last Known Good Time to storage: 2023-06-12T15:40:55
[1686666964.085916][20463:20503] CHIP:CTL: Joined the fabric at index 1. Fabric ID is 0x0000000000000001 (Compressed Fabric ID: A2C67A67B4C737DD)
[1686666964.085923][20463:20503] CHIP:IN: UDP::Init bind&listen port=5550
[1686666964.085937][20463:20503] CHIP:IN: UDP::Init bound to port=5550
[1686666964.085939][20463:20503] CHIP:IN: UDP::Init bind&listen port=5550
[1686666964.085948][20463:20503] CHIP:IN: UDP::Init bound to port=5550
[1686666964.085950][20463:20503] CHIP:IN: TransportMgr initialized
[1686667084.082815][20463:20463] CHIP:CTL: Shutting down the commissioner
[1686667084.082851][20463:20463] CHIP:CTL: Stopping commissioning discovery over DNS-SD
[1686667084.082924][20463:20463] CHIP:CTL: Shutting down the controller
[1686667084.082940][20463:20463] CHIP:IN: Expiring all sessions for fabric 0x1!!
[1686667084.082954][20463:20463] CHIP:FP: Forgetting fabric 0x1
[1686667084.082980][20463:20463] CHIP:TS: Pending Last Known Good Time: 2023-06-12T15:40:55
[1686667084.083126][20463:20463] CHIP:TS: Previous Last Known Good Time: 2023-06-12T15:40:55
[1686667084.083136][20463:20463] CHIP:TS: Reverted Last Known Good Time to previous value
[1686667084.083164][20463:20463] CHIP:CTL: Shutting down the commissioner
[1686667084.083170][20463:20463] CHIP:CTL: Stopping commissioning discovery over DNS-SD
[1686667084.083199][20463:20463] CHIP:CTL: Shutting down the controller
[1686667084.083212][20463:20463] CHIP:CTL: Shutting down the System State, this will teardown the CHIP Stack
[1686667084.083308][20463:20463] CHIP:DMG: All ReadHandler-s are clean, clear GlobalDirtySet
[1686667084.083340][20463:20463] CHIP:BLE: BleConnectionDelegate::CancelConnection is not implemented.
[1686667084.083357][20463:20463] CHIP:FP: Shutting down FabricTable
[1686667084.083366][20463:20463] CHIP:TS: Pending Last Known Good Time: 2023-06-12T15:40:55
[1686667084.083424][20463:20463] CHIP:TS: Previous Last Known Good Time: 2023-06-12T15:40:55
[1686667084.083435][20463:20463] CHIP:TS: Reverted Last Known Good Time to previous value
[1686667084.083618][20463:20463] CHIP:DL: writing settings to file (/mnt/chip_counters.ini-5gjrhI)
[1686667084.083900][20463:20463] CHIP:DL: renamed tmp file to file (/mnt/chip_counters.ini)
[1686667084.083927][20463:20463] CHIP:DL: NVS set: chip-counters/total-operational-hours = 0 (0x0)
[1686667084.083939][20463:20463] CHIP:DL: Inet Layer shutdown
[1686667084.083949][20463:20463] CHIP:DL: BLE shutdown
[1686667084.083962][20463:20463] CHIP:DL: System Layer shutdown
[1686667084.084161][20463:20463] CHIP:TOO: Run command failure: ../examples/chip-tool/commands/common/CHIPCommand.cpp:537: CHIP Error 0x00000032: Timeout
```

farshidtz commented:
@MonicaisHer Thanks for testing. I opened an issue: https://github.com/canonical/matter-pi-gpio-commander/issues/18
